From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: kbinani <kbinani.bt@gmail.com>
Date: Sun, 3 Dec 2023 21:44:40 +0900
Subject: [PATCH] Disable sysout warning


diff --git a/src/main/java/io/papermc/paper/logging/SysoutCatcher.java b/src/main/java/io/papermc/paper/logging/SysoutCatcher.java
index a8e813ca89b033f061e695288b3383bdcf128531..232e4e08e343859705abdfc5f59f7292dbc69fa1 100644
--- a/src/main/java/io/papermc/paper/logging/SysoutCatcher.java
+++ b/src/main/java/io/papermc/paper/logging/SysoutCatcher.java
@@ -32,8 +32,8 @@ public final class SysoutCatcher {
     private final ConcurrentMap<String, PluginNag> nagRecords = new ConcurrentHashMap<>(64);
 
     public SysoutCatcher() {
-        System.setOut(new WrappedOutStream(System.out, Level.INFO, "[STDOUT] "));
-        System.setErr(new WrappedOutStream(System.err, Level.SEVERE, "[STDERR] "));
+        // System.setOut(new WrappedOutStream(System.out, Level.INFO, "[STDOUT] "));
+        // System.setErr(new WrappedOutStream(System.err, Level.SEVERE, "[STDERR] "));
     }
 
     private final class WrappedOutStream extends PrintStream {
